### ISSUE TITLE
Use Wordpress comments for feedback

### ISSUE THREAD

**Issue by svenseeberg** [Integreat/cms#661]
Wordpress has an existing infrastructure for feedback -> comments

* Use a dummy post as "catch all" for comments?
* Can comments be transmitted via API?

http://v2.wp-api.org/reference/comments/ -> Go to "Create a Comment"

Additionally required fields:
* Enum('missing_search_result','comment_page',...)
* Rate page (1 to 5)

**Comment by maxammann** [comment]
Maybe we want an extended feedback with a thumb up or down.

**Comment by Schedulaar** [comment]
I would expect an endpoint like:
```
POST api.integreat-app.de/v3/feedback
{
	type: 		'category'  	| 'event'      | 'search'  | 'extra'
	id:   		'permlink'   	| 'event_id'   |   null    | 'key'
	comment: 	'comment' (optional)
	rating:		float (probably in [0,1] )
}
```

**Comment by timoludwig** [comment]
You can now post feedback the following way:
```
POST https://cms-test.integreat-app.de/augsburg/de/wp-json/wp/v2/comments
{
	'post': 3226,
	'content': {
		'type': 'category',
		'url': 'https://web.integreat-app.de/augsburg/de/erste-schritte',
		'comment': 'I like this category',
		'rating': 0.75,
		'some_other_custom_field': 'value'
	}
}
```
Please don't change the post 3226, because it is a dummy post to collect all feedback. Use a content-field to indicate the id or url of the post the feedback is referring to.

@maxammann @Schedulaar: Is this suitable for your needs?
@svenseeberg: Can we use the resulting comments, even though the content is rendered as html?

Apart from usability: The comments will show up on most administrator's dashboard, so we have to consider that this might confuse them.

**Comment by maxammann** [comment]
@timoludwig How much work would it be to use the integreat json api for this?

So the base path remains e.g. `api.integreat-app.de/v3/`
That would make it easier in the client as we only have to configure one api path.

also isn't there some redudant data like `post` and `url`? Or does this make it more convenient for us?

`url` should be called  `permalink` probably.

**Comment by timoludwig** [comment]
If we use the commenting-infrastructure of wordpress, it is no problem to provide an additional feedback-endpoint via the APIv3. If we have to use something else, e.g. a custom post-type or an additional sql table as well as the backend-functionality to manage the feedback, it is quite a lot of work to do.

If we use the default wordpress-api like described above, we need `'post': 3226` for every request, no matter which id the actual post has. Hence, the id or permalink information in the content-field is not redundant. And as all content-fields are stored in one comment, the attributes' names are completely up to you (you can even pass additional fields if you want to)...

**Comment by maxammann** [comment]
I think we definitely should use the wordpress commenting functionality. Maybe we can proxy that call through our api v3.

We also need an endpoint for comments on searches (Like: Your search is empty. What would you expect here?) and maybe extras.

**Comment by timoludwig** [comment]
Ok. In my opinion, it makes sense to indicate the type ("page", "event", "category", "search", "extra") by a type-parameter of the feedback endpoint. So we don't need that many new endpoints and we are more flexible in case there are new feedback-types in the future.

**Comment by maxammann** [comment]
Sounds good if we can set any type :) :+1:

**Comment by timoludwig** [comment]
was solved in #744
